### PR TITLE
change collection context to normalizer

### PR DESCRIPTION
**Pull request by auamarto**

### CODE REVIEW

File: Serializer/Serializer.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Opstalent\ApiBundle\Serializer;
+
+use \Symfony\Component\Serializer\Serializer as BaseSerializer;
```
Comment (patgrudniewski) [inline Serializer/Serializer.php]: remove first slash. it's unnecessary

File: Serializer/Serializer.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ * {@inheritDoc}
+ */
+class Serializer extends BaseSerializer
+{
+    /**
+     * @param array $context
+     * @param $data
+     * @return array
+     */
+    protected function buildContext(array $context = [], $data)
```
Comment (patgrudniewski) [inline Serializer/Serializer.php]: return typehint

File: Serializer/Serializer.php [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    {
+        if(array_key_exists("top", $context)
+            && is_object($data)
+        ) {
+            unset($context["top"]);
+            $context['entity'] = $data;
```
Comment (patgrudniewski) [inline Serializer/Serializer.php:23]: can you explain this logic?

**Review by patgrudniewski — CHANGES_REQUESTED** [review]

File: Serializer/Serializer.php (hunk 3 above)
Comment (auamarto) [reply Serializer/Serializer.php:23]: it take first object (so it will be collection entity for list) as cotnext

**Review by patgrudniewski — APPROVED** [review]

**Pull request merged**